JustSaying.Models/JustSaying.Models.csproj

```diff
@@ -6,4 +6,7 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="SourceLink.Embed.AllSourceFiles" Version="2.8.1" />
```

Comment:
If you put this into `Directory.Build.props` it will apply to every project in the solution.

Reply:
I didn't add that, so the tools must have. Should we move it?

Reply:
Yep, it needs to go in `Directory.Build.props` since the old version is there. Tooling has made a boo-boo.